app/models/manageiq/providers/google/inventory/parser/network_manager.rb

```diff
@@ -350,7 +350,7 @@ def load_balancer_health_check_members(persister_lb_health_check, target_pool)
         :status_reason              => ""
       )
     end
-  rescue Fog::Errors::Error => err
+  rescue Fog::Errors::Error, Google::Apis::ClientError => err
```

Comment:
It looks like we already were handling this case but a newer version of the gem raises a different exception

Reply:
Forgive my ignorance, but wasn't one of reasons to use fog to begin with so that we wouldn't have to worry about gem-specific exceptions like this? Is there a newer version of fog-google perhaps?

Reply:
Honestly for MIQ providers the abstraction that Fog provides really isn't helpful.  Having a common Compute Vm definition across multiple cloud providers isn't something that we need because our providers are already specific to a provider type.